LinqFaster/Select.cs

```diff
@@ -245,12 +143,12 @@ public static void SelectInPlaceF<T>(this List<T> source, Func<T, int, T> select
         {
             if (source == null)
             {
-                throw Error.ArgumentNull("source");
+                throw Error.ArgumentNull(nameof(source));
```

Comment:
isolating the changes would be a major help to reviewers. These, while potentially desirable, provide a lot of noise as they aren't "Convert SumF into a generic function"